tests/gabc-dump/glyphs/above_lines_text.gabc

```diff
@@ -1,4 +1,4 @@
 name: above lines text;
 %%
-gh<alt>above lines</alt>(ghZ)
-gh(gh[alt:note alt]Z)
+gh<alt>above lines $(parentheses)</alt>(ghZ)
```

Comment:
Are the closing parentheses not escaped?

Reply:
No, they are not.  Lexically, close parenthesis does not have special meaning in ordinary text.